pupil-spa/src/app/privacy/privacy.component.html

```diff
@@ -0,0 +1,184 @@
+        restricted to those STA and OliveJar Digital employees who need to see the survey data. All data for the survey
+         will be supplied in electronic data files and sent by secure file transfer protocol.
+      </p>
+
+      <p>
+        No information gathered during the trial will be transferred or stored outside of the European Economic Area (EEA). 
+        OliveJar Digital handles personal data in accordance with the rights given to individuals under UK data protection legislation.
+         Data subjects have a right to see information held about them and OliveJar Digital will cooperate fully when a 
+         subject access request is made of the data controller (in this instance, STA).
+      </p>
+
+      <h2 class="heading-small heading-contents" id="retain">Retaining personal data</h2>
+
+      
+      <p>
+        Data collected by OliveJar Digital will be retained for up to 6 months following the end of the survey, as agreed by STA and OliveJar Digital.
+         On STA’s approval, OliveJar Digital will destroy data and permanently delete computer files from their systems.
+      </p>
+
+      <p>
+          STA will only keep personal data for as long as it is needed for the purposes of this project, 
+          after which point it will be securely destroyed. Please note that, under section 33 of the Data Protection Act 1998,
+          and in compliance with the relevant conditions, we can lawfully keep personal data processed purely for 
+          research purposes indefinitely.
+      </p>
+      <h2 class="heading-small heading-contents" id="info">Further information</h2>
+
+      
+      <p>
+        If you have any queries about MTC trial data, please email <a href="mailto:MTC.STA@education.gov.uk">MTC.STA@education.gov.uk</a>.
+      </p>
+
+      <p>
+        STA, as an executive agency of the Department for Education (DfE), determines the purposes and means 
+        of processing personal data as part of this project. You can find more information in the DfE’s 
+        <a target="_blank" rel="noopener noreferrer" href="https://www.gov.uk/government/organisations/department-for-education/about/personal-information-charter">
+          personal information charter
+        </a>.
+      </p>
+
+      <p>
+        You have the right to lodge a complaint about data protection issues with the Information Commissioner’s Office. 
+        More information is available on the <a target="_blank" rel="noopener noreferrer" href="https://www.ico.org.uk/concerns/">ICO website</a>.
+      </p>
+
+      <p>© Crown copyright 2018</p>
+
+      <p class="top">
+        <a href="/ict-survey/privacy#content">Back to top</a>
+      </p>
+    </div>
+
+    <div class="column-one-third related">
+      <div class="border"></div>
+      <h2 class="heading-small heading-contents">Related</h2>
+      <a href="{{linkCookies}}">Cookies</a>
+    </div>
+
```

Comment:
@danwilliams why soo much whitespace?

Reply:
easier to read IMO

Reply:
@GuyHarwood I don't see the point for having 3 empty lines when 1 should be enough.
It's a non blocker really but just wanted to point it out.

Reply:
I've tweaked this slightly, comments were fair 👍